custom_components/icsee_ptz/asyncio_dvrip.py

```diff
@@ -1,16 +1,18 @@
+import asyncio
+import hashlib
+import json
+import logging
 import os
 import struct
-import json
```

Comment:
This file comes directly from https://github.com/OpenIPC/python-dvr/blob/master/asyncio_dvrip.py
I made some small change (forgot what it was exactly).
Therefore, it's better not to refactor it so we can still diff for changes with the original if we ever update it in the future

Reply:
Can you make the linter ignore this file? Then I can merge your other workflows pr :)